enderio-base/src/main/java/crazypants/enderio/base/filter/INetworkedInventory.java

```diff
@@ -1,84 +0,0 @@
-package crazypants.enderio.base.filter;
```

Comment:
Why? This was a quite new interface...

Reply:
The interface only existed so that filters could be ported super easily, but now that they go straight to IItemHandler I didn't see the need for the interface. I can make a better one to put in base or conduits as needed though as the old one wasn't that great honestly.

Reply:
Just wanted to hear your reasoning as we're generally moving toward *more* interfaces to allow looser coupling.

Reply:
Thats fair, I like the idea of the interface, however it wasn't adding anything at the time. I want to write a new one when I have a better chance to clean up the way NetworkedInventory works